README.md

```diff
@@ -56,21 +57,23 @@ interface Props {
   controls?: boolean;
   autoPlay?: boolean;
   preload?: string;
+  textTracks?: VideoTextTracks;
 }
 ```
 
-**render method**
+### Render method
 
 ```typescript
-type RenderCallback = (videoElement: ReactNode, state: VideoState, actions: VideoActions) => ReactNode;
+type RenderCallback = (reactElement: ReactElement<HTMLMediaElement>, state: VideoState, actions: VideoActions, ref: React.RefObject<HTMLMediaElement>) => ReactNode;
```

Comment:
For a second I thought we never had a `ref`... and nothing made sense, later I realized docs where not sync! 
Thanks!!